bloom-housing/bloom
# Add order/orderBy array support for listings query params

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description

This CR changes listings query params interface to support order/orderBy array param. Sorting by multiple keys is now possible in /listings endpoint. 

## How Can This Be Tested/Reviewed?

Reviewing this requires using postman or other HTTP client tool, check if e.g. following query works for you and the returned listings order is correct: 

```
{{baseUrl}}/listings?orderBy[0]=status&order[0]=ASC&limit=2&view=full&orderBy[1]=mostRecentlyUpdated&order[1]=ASC
```

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
